src/commcare_cloud/fab/fabfile.py

```diff
@@ -550,13 +560,14 @@ def unlink_current():
         sudo('unlink {}'.format(env.code_current))
 
 
-def copy_release_files():
-    execute(release.copy_localsettings)
-    execute(release.copy_tf_localsettings)
-    execute(release.copy_formplayer_properties)
-    execute(release.copy_components)
-    execute(release.copy_node_modules)
-    execute(release.copy_compressed_js_staticfiles)
+def copy_release_files(manage=False):
```

Comment:
The name of this arg is a bit meaningless here. Maybe rename to something like `full_cluster=True`?